src/main/java/com/revature/domain/QuestionComment.java

```diff
@@ -0,0 +1,101 @@
+package com.revature.domain;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.io.Serializable;
+
+@Entity
+@Table(name="ie_question_comment")
+public class QuestionComment implements Serializable {
+
+	@Id
+	@Column(name="qc_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="questionCommentSeq")
+	@SequenceGenerator(allocationSize=1, name="questionCommentSeq", sequenceName="question_comment_seq")
+	private Integer id;
+	
+	@Column(name="qc_comment_text")
+	private String commentText;
+	
+	@ManyToOne(fetch=FetchType.EAGER)
+	@JoinColumn(name="qc_eid")
+	private QuestionEval eval;
+
+	public QuestionComment(){}
+
+	public QuestionComment(String commentText, QuestionEval eval) {
+		super();
+		this.commentText = commentText;
+		this.eval = eval;
+	}
+
+	public Integer getId() {
+		return id;
+	}
+
+	public void setId(Integer id) {
+		this.id = id;
+	}
+
+	public String getCommentText() {
+		return commentText;
+	}
+
+	public void setCommentText(String commentText) {
+		this.commentText = commentText;
+	}
+
+	@JsonIgnore
+	public QuestionEval getEval() {
+		return eval;
+	}
+
+	public void setEval(QuestionEval eval) {
+		this.eval = eval;
+	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((commentText == null) ? 0 : commentText.hashCode());
+//		result = prime * result + ((eval == null) ? 0 : eval.hashCode());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)